tkinter.test.widget.AbstractWidgetTest has two tests skip for Scale.  I removed the skip in test_keys that works with this patch.  The skipped part of test_param,
```
        if not isinstance(widget, Scale):
            t = widget.configure(name)
            self.assertEqual(len(t), 5)
            self.assertEqual2(t[4], expected, eq=eq)
```
still fails after the patch, 9 times, at the configure call because the override tries to update kw with a string.
```
Traceback (most recent call last):
  File "f:\dev\3x\lib\tkinter\test\widget_tests.py", line 497, in test_variable
    self.checkVariableParam(widget, 'variable', var)
  File "f:\dev\3x\lib\tkinter\test\widget_tests.py", line 195, in checkVariableParam
    self.checkParam(widget, name, var, conv=str)
  File "f:\dev\3x\lib\tkinter\test\widget_tests.py", line 67, in checkParam
    t = widget.configure(name)
  File "f:\dev\3x\lib\tkinter\ttk.py", line 1088, in configure
    kw.update(cnf)
ValueError: dictionary update sequence element #0 has length 1; 2 is required
```
The solution is to not do this.  I will submit a further commit that fixes this and removes the 2nd skip.

--- a/Lib/tkinter/test/widget_tests.py
+++ b/Lib/tkinter/test/widget_tests.py
@@ -3,7 +3,6 @@
 import unittest
 import sys
 import tkinter
-from tkinter.ttk import Scale
 from tkinter.test.support import (AbstractTkTest, tcl_version, requires_tcl,
                                   get_tk_patchlevel, pixels_conv, tcl_obj_eq)
 import test.support
@@ -63,11 +62,9 @@ class AbstractWidgetTest(AbstractTkTest):
             eq = tcl_obj_eq
         self.assertEqual2(widget[name], expected, eq=eq)
         self.assertEqual2(widget.cget(name), expected, eq=eq)
-        # XXX
-        if not isinstance(widget, Scale):
-            t = widget.configure(name)
-            self.assertEqual(len(t), 5)
-            self.assertEqual2(t[4], expected, eq=eq)
+        t = widget.configure(name)
+        self.assertEqual(len(t), 5)
+        self.assertEqual2(t[4], expected, eq=eq)
 
     def checkInvalidParam(self, widget, name, value, errmsg=None, *,
                           keep_orig=True):
@@ -209,7 +206,6 @@ class AbstractWidgetTest(AbstractTkTest):
     def test_keys(self):
         widget = self.create()
         keys = widget.keys()
-        # XXX
         self.assertEqual(sorted(keys), sorted(widget.configure()))
         for k in keys:
             widget[k]

--- a/Lib/tkinter/ttk.py
+++ b/Lib/tkinter/ttk.py
@@ -1084,9 +1084,9 @@ class Scale(Widget, tkinter.Scale):
 
         Setting a value for any of the "from", "from_" or "to" options
         generates a <<RangeChanged>> event."""
-        if cnf:
+        retval = Widget.configure(self, cnf, **kw)
+        if not isinstance(cnf, (type(None), str)):
             kw.update(cnf)
-        retval = Widget.configure(self, **kw)
         if any(['from' in kw, 'from_' in kw, 'to' in kw]):
             self.event_generate('<<RangeChanged>>')
         return retval

--- a/Misc/NEWS.d/next/Library/2020-01-03-18-02-50.bpo-39152.JgPjCC.rst
+++ b/Misc/NEWS.d/next/Library/2020-01-03-18-02-50.bpo-39152.JgPjCC.rst
@@ -1,2 +1,2 @@
-Add missing ttk.Scale.configure return value.  ttk.Scale().configure() now
-returns a configuration.  Change posted by Giovanni Lombardo.
+Fix ttk.Scale.configure([name]) to return configuration tuple for name
+or all options.  Giovanni Lombardo contributed part of the patch.

